k8s/overlays/coreso/test/swe-gridcapa-runner-itools-coreso-configmap.yaml

```diff
@@ -47,13 +42,13 @@ data:
       relative-mip-gap: 0.001
 
     rao-topological-actions-optimization:
-      max-search-tree-depth: 2
+      max-search-tree-depth: 3
       relative-minimum-impact-threshold: 0.0
       absolute-minimum-impact-threshold: 1.0
 
     rao-second-preventive-rao:
       execution-condition: POSSIBLE_CURATIVE_IMPROVEMENT
-      re-optimize-curative-range-actions: true
+      re-optimize-curative-range-actions: false
```

Comment:
I don't know why this parameter was false in Azure test (I think it's an error). We should keep it true for Coreso and change it in the itools swe azure